tests/ISA/ISA.Tests/ArcTable.Tests.fs

```diff
@@ -2186,6 +2186,17 @@ let private tests_Join = testList "Join" [
     ]
 ]
 
+let private tests_IterColumns = testList "IterColumns" [
+    testCase "Replace input column header" <| fun _ ->
+        let table = create_testTable()
+        let expected = CompositeHeader.Input IOType.RawDataFile
```

Comment:
The `IterColumns` actually allows mutation of the underlying values? Doesn't the `this.GetColumn` return a copy?

Reply:
Nvm you use Iteri and update by index